index.md

```diff
@@ -7,8 +7,11 @@ redirect_from: /lang/en/
 
 ## Summary
 
-As a software developer, I want to squash feature branches onto `master` and write
-a standardized commit message while doing so.
+As a software developer, I want to create a readable commit history writing standardized commit messages to make easier and explicit what changes have been applied to a project.
+To accomplish it, the Conventional Commits specification proposes introducing a standardized lightweight
```

Comment:
> proposes introducing a standardized

I'll be honest, I never really understood the phrase `proposes introducing a ...`. From my perspective, the existence of this document means that the standard/convention has already been introduced. No need to _propose_ introducing the standard/convention now.

Maybe: `... the Conventional Commits specification is a lightweight convention on top of ...`

Reply:
Yeah, it makes sense. 👍